source/CinematicCamera/Modules/CinematicCamera/ModuleData/module_strings.xml

```diff
@@ -37,5 +37,7 @@
   <string id="str_key_name.CinematicCameraHotKey_InitiateTeamChat" text="{=CinematicCamera_increase_fov}Increase FOV" />
   <string id="str_key_description.CinematicCameraHotKey_InitiateTeamChat" text="{=CinematicCamera_increase_fov_hint}Increase Field of View." />
   <string id="str_key_name.CinematicCameraHotKey_FinalizeChat" text="{=CinematicCamera_decrease_fov}Decrease FOV" />
-  <string id="str_key_description.CinematicCameraHotKey_FinalizeChat" text="{=CinematicCamera_decrease_fov_hint}Decrease Field of View." />
+  <string id="str_key_description.CinematicCameraHotKey_FinalizeChat" text="{=CinematicCamera_decrease_fov_hint}Decrease Field of View." />  
```

Comment:
Maybe we need to also modify std_CinematicCamera.xml and CinematicCamera.zho-CN.xml. I will do that later.